mods/sp/audio/voices.yaml

```diff
@@ -444,9 +444,10 @@ Deva:
 		Die: devaquote0003, devaquote0004, devaquote0005
 
 Tripod:
+	DefaultVariant: .wav
 	Voices:
 		Select: TripSound0001, TripSound0002, TripSound0003
 		Move: TripSound0003, TripSound0004, TripSound0005
-		Attack: TripSound0006, TripSound0007, TripSound0008
-		Action: TripSound0006, TripSound0007, TripSound0008
-		Die: TripSound0011, TripSound0012
\ No newline at end of file
+		Attack: TripSound0006, TripSound0007, TripSound0008, TripSound0009, TripSound00010
+		Action: TripSound0006, TripSound0007, TripSound0008, TripSound0009, TripSound00010
+		Die: TripSound0011, TripSound0012
```

Comment:
You could also fix the typo causing the filenames to lack a leading zero here if you(r tool) already adds the closing newline.